frontend/src/pages/vault/stake/Stake.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import STRK from '../../../assets/icons/strk.svg';
+import USDC from '../../../assets/icons/borrow_usdc.svg';
+import './stake.css';
+import { VaultLayout } from '../VaultLayout';
+import { CogIcon } from 'lucide-react';
+
+function Stake() {
+  const [selectedNetwork, setSelectedNetwork] = useState('Starknet');
+
+  const networks = [{ name: 'Starknet', image: STRK }];
+
+  const handleChange = (e) => {
+    setSelectedNetwork(e.target.value);
+  };
+  return (
+    <VaultLayout>
+      <div className="staking-container">
+        <div className="balance-row">
+          <div className="balance-card">
+            <div className="balance-card-header">
+              <img src={STRK} alt="STRK" className="currency-icon" />
+              <span className="balance-label">STRK Balance</span>
+            </div>
+            <span className="balance-amount">0.046731</span>
+          </div>
+          <div className="balance-card">
```

Comment:
please make sure these cards are done according to figma, currently there are some mismatches in aligning with icons and font sizes

Reply:
still undersolved